package.json

```diff
@@ -216,7 +216,7 @@
     "universal-ga": "^1.2.0",
     "url": "^0.11.0",
     "url-loader": "^0.5.7",
-    "victory": "0.27.0",
+    "victory": "^30.0.0",
```

Comment:
better to make this `30.0.0` rather than `^`

Reply:
i looked into this more.  it is so confusing that i'm going to just pin it as adam suggested.